sidecarhttp/envoy_api.go

```diff
@@ -3,6 +3,7 @@ package sidecarhttp
 //go:generate ffjson $GOFILE
 
 import (
+	"encoding/json"
```

Comment:
We really don't want to do that. The CPU overhead and GC pressure are MUCH higher with encoding/json instead of ffjson.

Reply:
@relistan Correct, it was easier for the PoC to disable `ffjson` for me. I plan to enable it again according the `TODO` entries.